src/DonationForms/resources/registrars/templates/fields/Amount/DonationAmountLevels.tsx

```diff
@@ -23,24 +24,51 @@ export default function DonationAmountLevels({
     const amount = useWatch({name});
     const formatter = useCurrencyFormatter(currency);
 
+    const groupedLevels = levels.reduce(
+        (acc, level) => {
+            const key = level.label ? 'labeled' : 'unlabeled';
+            acc[key].push(level);
+            return acc;
+        },
+        {labeled: [], unlabeled: []}
+    );
+
+    const allLevels = [...groupedLevels.labeled, ...groupedLevels.unlabeled];
```

Comment:
@pauloiankoski it looks like these are untyped which could cause some confusion during the implementation below since we were previously using the types levels array.  If you add types for `labeled` and `unlabeled` it should result in type definitions for allLevels

Reply:
Resolved: 3a5fc46567692475a0c01bed07c840421ceae193